admin/qtx_admin_settings.php

```diff
@@ -743,7 +744,17 @@ class="qtranxs_explanation"><?php _e( 'The built-in integration modules are auto
                         foreach ( $modules as $module ):
                             ?>
                             <tr>
-                                <td><?php echo $module['name'] ?></td>
+                                <td>
+                                    <?php if ( isset( $q_config['ma_module_enabled'][ $module['id'] ] ) ) : ?>
+                                        <label for="ma_module_enabled_<?php echo $module['id']; ?>">
+                                            <input type="checkbox"
+                                                   name="ma_module_enabled[<?php echo $module['id']; ?>]"
+                                                   id="ma_module_enabled_<?php echo $module['id']; ?>"
+                                                   value="1"<?php checked( $q_config['ma_module_enabled'][ $module['id'] ] ) ?>/>
+                                            <?php echo $module['name']; ?>
+                                        </label>
+                                    <?php else: echo $module['name']; endif; ?>
+                                </td>
```

Comment:
Good for now in but eventually I'd want a checkbox for all modules. But some have dependencies, I'll look how to evolve this.

Reply:
As long as it's only a matter of dependencies it should be enough to add `'manual_activation' => true` flag in configuration and it should work, the option will be just an additional condition to the dependencies (needs to be tested though).
But as those are integration plugin, maybe some of them are strictly needed for the relevant plugin to work correctly or at all with qtranslate, I don't know. If this is the case that should not be a setting exposed to the user.
If it's just a matter of layout, greyed-out checkboxes can be easily added following the internal plugin state.

Reply:
There's some bug, I tried adding `manual_activation` to ACF just to try, and the checkbox doesn't appear.
If you know of a quick fix that would be good.

Looking forward, this looks too complicated to maintain, I'd like to simplify this without `ma_module_enabled` in `q_config`. This info should be better integrated with the `$module` object. I didn't expect so many changes regarding the modules before the next release.

Reply:
pls check #1147